zaza/openstack/charm_tests/tempest/templates/tempest_v3.j2

```diff
@@ -77,10 +77,29 @@ port_security = true
 {% endif %}
 
 {% if 'heat' in enabled_services %}
-[orchestration]
-stack_owner_role = Admin
-instance_type = m1.small
+[heat_plugin]
 keypair_name = testkey
+username = demo
+password = password
+project_name = demoProject
+user_domain_name = demoDomain
+project_domain_name = demoDomain
+admin_username = {{ admin_username }}
+admin_password = {{ admin_password }}
+admin_project_name = {{ admin_project_name }}
+admin_user_domain_name = admin_domain
+admin_project_domain_name = admin_domain
+auth_url = {{ proto }}://{{ keystone }}:5000//v3
+auth_version = 3
+disable_ssl_certificate_validation = true
```

Comment:
if we are disabling cert validation, we can probably get away with not setting the `ca_file` config key, it will allow us to not rely where the root CA is.

Reply:
we probably need to set this to true too - https://github.com/openstack/heat-tempest-plugin/blob/master/heat_tempest_plugin/config.py#L170

Reply:
Thanks for taking a look. I've got a new version with cleanup incoming. I'd like to enable the certs eventually once the old bundles that don't use vault are no longer used.